src/Blocks/custom/form/components/form-mailerlite-options.js

```diff
@@ -1,41 +1,39 @@
-import _ from 'lodash';
 import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
 import { SelectControl } from '@wordpress/components';
+import { getAttrKey } from '@eightshift/frontend-libs/scripts/helpers';
+import manifest from '../manifest.json';
 
 /**
  * Options component
  *
  * @param {object} props Component props.
  */
 export const FormMailerliteOptions = (props) => {
-  const {
-    groups,
-    groupId,
-    onChangeGroupId,
-  } = props;
+	const {
+		attributes,
+		setAttributes,
+		formMailerliteGroupId,
+		formMailerliteGroups,
+	} = props;
 
-  const groupsOptions = groups.length && [
-    {
-      value: '',
-      label: __('Please select group', 'eightshift-forms'),
-    },
-    ...groups,
-  ];
+	const groupsOptions = formMailerliteGroups.length && [
```

Comment:
options in manifest and getOption

Reply:
Same as above, defined in your project in PHP and then passed to form via `window.eightshiftForms`